### PR TITLE
[WM-2500][WM-2502] Pass ECM base URL to Cromwell pods

### PR DESCRIPTION
**Pull request by salonishah11**
__Jira ticket__: https://broadworkbench.atlassian.net/browse/WM-2500, https://broadworkbench.atlassian.net/browse/WM-2502

### What

- Adds support for setting ECM url in Cromwell pods

### Why

- To support private workflows

## Testing these changes

Tested the changes in a BEE along with terra-helmfile changes ([see this PR](https://github.com/broadinstitute/terra-helmfile/pull/5313)) and verified the boolean values and ECM base url being set as expected in Cromwell  pods configs. For example, this is from Cromwell Runner config
```
...
GithubAuthVending {
        config {
          enabled = true
          auth.azure = true
          ecm.base-url = "https://externalcreds.dsde-dev.broadinstitute.org"
        }
      }
...
```

### Who tested and where

- [x] This change is covered by automated tests 
  - _NB: Rerun automation tests on this PR by commenting `jenkins retest` or `jenkins multi-test`._
- [x] I validated this change 
- [ ] Primary reviewer validated this change 
- [ ] I validated this change in the dev environment

### CODE REVIEW

**Review by kpierre13 — APPROVED** [review]

File: http/src/main/resources/leo.conf [hunk 1]
```diff
@@ -198,6 +198,7 @@ azure {
   cromwell-runner-app-config {
     instrumentation-enabled = ${?CROMWELL_RUNNER_APP_INSTRUMENTATION_ENABLED}
     enabled = ${?CROMWELL_RUNNER_APP_ENABLED}
+    ecm-base-uri = ${?ECM_URL}
```
Comment (LizBaldo) [inline http/src/main/resources/leo.conf:201]: 🤔 where will the URL come from for non BEE environments? It seems like you also need to specify it for the live environments in this PR https://github.com/broadinstitute/terra-helmfile/pull/5313/files

Comment (salonishah11) [reply http/src/main/resources/leo.conf:201]: That is a valid question and I should have linked another PR in here. Apologies for that! We did add the ecm-base-url for other environments in a previous PR - https://github.com/broadinstitute/terra-helmfile/pull/5236/files

Comment (LizBaldo) [reply http/src/main/resources/leo.conf:201]: Ah perfect! In that case this looks good :) I will 👍

**Review by LizBaldo — APPROVED** [review]
LGTM, thanks 👍

**Pull request merged**